contracts/pallet-ibc/src/lib.rs

```diff
@@ -266,6 +267,9 @@ pub mod pallet {
 		type IsReceiveEnabled: Get<bool>;
 
 		type FeeAccount: Get<Self::AccountIdConversion>;
+
+		#[pallet::constant]
+		type ServiceCharge: Get<Perbill>;
```

Comment:
and there is ServiceCharge in fee pallet? can we have one for two?

Reply:
what if we want to have different settings for fee 
- one of them for charing when asset delivered to the chain.
- when assets transferred from the chain
for example : 
- 0.04% when in
- 0.09% when out

Reply:
then lets rename to clarify that, ServiceChargeIn and ServiceChargeOut